src/lib/rest/rest.cpp

```diff
@@ -1435,7 +1441,16 @@ static int restStart(IpVersion ipVersion, const char* httpsKey = NULL, const cha
 
   if (threadPoolSize != 0)
   {
-    serverMode = MHD_USE_SELECT_INTERNALLY;
+    //
+    // To use poll() instead of select(), MHD 0.9.48 has the define MHD_USE_EPOLL_LINUX_ONLY,
+    // while in MHD 0.9.51, the name of the define has changed to MHD_USE_EPOLL.
+    // So, to support both names, we need a ifdef/else cpp directive here.
+    //
+#ifdef MHD_USE_EPOLL
```

Comment:
Looks smart, as this will work if we some day upgrade MHD. However, it wouldn't be a problem also to "stick" to 0.9.48 now and change the code in the future :)

NTC